### ISSUE TITLE
ONNX Export Issue

### ISSUE THREAD

**Issue by TimJVK** [sassoftware/python-dlpy#311]
I am using the functional API to build a very basic network:

`all_inputs ='C_Ref_1','C_Ref_2','C_Ref_3','C_Ref_4','C_Ref_5','C_Ref_6','C_Ref_7','C_Ref_8','C_Ref_9','C_Ref_10','C_Ref_11','C_Ref_12','C_Ref_13','C_Ref_14','C_Ref_15'`

Build model with DPLy functional API:
`input1 = Input(n_channels=1,std='MIDRANGE')`
`dense1 = Dense(n=50,act='tanh')(input1)`
`output1 = OutputLayer(act='auto', n=1)(dense1)`
`output2 = OutputLayer(act='auto', n=1)(dense1)`
`model_functional = Model(conn = conn, inputs = input1, outputs = [output1, output2])`
`model_functional.compile()`

Train Model:
`model_functional.fit(data=conn.CASTable("_PARTITIONEDDATA_"), `
`                     seed=44444444,`
`                     n_threads=4, `
`                     data_specs=[DataSpec(type_='NUMERICNOMINAL',`
`                                         layer='InputLayer_1',`
`                                         data=all_inputs`
`                                        ),`
`                                 DataSpec(type_='NUMERICNOMINAL',`
`                                         layer='OutputLayer_1',`
`                                         data='out1'`
`                                        ),`
`                                 DataSpec(type_='NUMERICNOMINAL',`
`                                         layer='OutputLayer_2',`
`                                         data='out2'`
`                                        )`
`                                ],`
`                     optimizer=Optimizer(algorithm=AdamSolver(beta1=0.90945, beta2=0.90945, learning_rate=0.0000707946),`
`                                         dropout=0,`
`                                         mini_batch_size=5, `
`                                         max_epochs=250`
`                                        )`
`                    )`

When attempting to export ONNX model:
![image](https://user-images.githubusercontent.com/44534207/86805660-16252c00-c046-11ea-9838-93e2e6f62030.png)

Since I am passing tabular data to the input layer, do I need to define dimensions for the layer?

**Comment by eusdougc** [comment]
@TimJVK - my apologies for the delayed response.  We will look into this and get back to you.

**Comment by eusdougc** [comment]
@TimJVK - when you build your model, please change this line

input1 = Input(n_channels=1,  std='MIDRANGE')

to this

input1 = Input(n_channels=1, height=1, width=15, std='MIDRANGE')

That should solve your problem.

**Comment by TimJVK** [comment]
That resolved the issue, thank you!
